 auto.config.ts

```diff
@@ -7,7 +7,7 @@ import type { AutoRc } from 'auto';
 export default function rc(): AutoRc {
   return {
     baseBranch: 'main',
-    plugins: ['npm', 'released', ['exec', { afterShipIt: 'yarn publish-action' }]],
```

Comment:
This turned out to be unpredictable and hard to debug, so I just updated the publish-action script to always run on release and figure out the proper tag from the version in package.json.